src/components/AnalysisContent/Actions/DownloadPDF.js

```diff
@@ -144,26 +149,25 @@ const createPdf = (Document, Image, Link, Page, Text, View) => {
           </View>
           {topic === 'topic' ? (
             <View style={classes.section}>
-              {data.content.body.map(c => {
-                if (c.type === 'text') {
-                  return c.value.split('</p>').map(t => (
-                    <Text key={t} style={classes.text}>
-                      {t.replace(/<(?:.|\n)*?>/gi, '')}
-                    </Text>
-                  ));
-                }
-                return null;
-              })}
+              {getPlainText(data.content.content).map(t => (
+                <Text key={t} style={classes.text}>
+                  {t.replace(/<(?:.|\n)*?>/gi, '')}
+                </Text>
+              ))}
             </View>
           ) : (
             <View style={classes.section}>
-              <Text style={classes.boldText}>
-                {data.item.name}, {data.item.title}
-              </Text>
-              {data.item.description.split('</p>').map(t => (
-                <Text style={classes.text}>
-                  {t.replace(/<(?:.|\n)*?>/gi, '')}
-                </Text>
+              {data.item.map(c => (
+                <>
+                  <Text style={classes.boldText}>
+                    {c.carousel_name}, {c.carousel_title}
```

Comment:
Lets also check here if `c.carousel_name` actually exists before adding the comma... Institutions don't have names, just titles
![Screenshot from 2019-10-16 14-03-20](https://user-images.githubusercontent.com/1779590/66913637-bd2be880-f01d-11e9-91e3-a7a611a06783.png)